rules/do-you-know-how-to-claim-expense-reimbursements-2/rule.md

```diff
@@ -3,27 +3,23 @@ type: rule
 archivedreason: 
 title: Do you know how to claim expense reimbursements with Xero Expense app
 guid: 2e71cd53-d456-45bc-8a89-5ad9948cd85b
-uri: do-you-know-how-to-claim-expense-reimbursements-2
+uri: do-you-know-how-to-claim-expense-reimbursements-with-Xero-Expense-App
```

Comment:
It looks like this was an old url before, why was it changed to begin with?

Reply:
As per our conversation, there is another url for 'Do you know how to claim expense reimbursements'. I didn't want to change that URL and have it mixed up, so I decided to change it to 'with Xero Expense App'. We are not sure why historically there was a change from 'With Xero Expense App' to '-2', however, we are not going to look into it at this stage.